iceoryx_posh/test/moduletests/test_posh_runtime.cpp

```diff
@@ -135,8 +135,19 @@ TEST_F(PoshRuntime_test, LeadingSlashAppName)
 {
     const iox::RuntimeName_t invalidAppName = "/miau";
 
-    EXPECT_DEATH({ PoshRuntime::initRuntime(invalidAppName); },
-                 "Cannot initialize runtime. Please remove leading slash from Application name /miau");
+    auto errorHandlerCalled{false};
+    iox::Error receivedError{iox::Error::kNO_ERROR};
+    auto errorHandlerGuard = iox::ErrorHandler::SetTemporaryErrorHandler(
```

Comment:
You are right. I removed this test case from the fixture like in the test case below. IMHO that's better than a death test because we check for the actual error.